src/standalone/Vegetation/Canopy.jl

```diff
@@ -543,7 +558,7 @@ A helper function which returns the surface height for the canopy
 model, which is stored in the parameter struct.
 """
 function ClimaLSM.surface_height(model::CanopyModel, _...)
-    return model.parameters.h_c
+    return model.hydraulics.compartment_surfaces[end]
```

Comment:
I think we should add a comment in the `PlantHydraulicsModel` struct explaining that the height of the canopy is the last element of the `compartment_surfaces` struct, since that might not be very obvious.